plugins/announcements/src/index.ts

```diff
@@ -15,3 +16,8 @@ export type AnnouncementsApi = AnnouncementsApi_;
  * @deprecated Use `announcementsApiRef` from `@procore-oss/backstage-plugin-announcements-react` instead
  */
 export const announcementsApiRef = announcementsApiRef_;
+
+/**
+ * @public
+ */
+export const announcementsTranslationRef = translationRef_;
```

Comment:
Revert this change, and things look good to me. The other exports were migrated to `announcements-react` and are marked as `@deprecated`.

Reply:
Do you mean to revert the 'public' part, or the full export?

If its to revert the export too, I'd have to install locally the announcements-react too to use the translation, doing it here, i could install only the @procore-oss/backstage-plugin-announcements and be able to translate.

Reply:
> Do you mean to revert the 'public' part, or the full export?
> 

The full export. It needs to come from `announcements-react`. 

> If its to revert the export too, I'd have to install locally the announcements-react too to use the translation

Installing `announcements-react` to use this is the right path though. It's the recommended way from Backstage and its how they have their core plugins designed. Take a look at the catalog or scaffolder plugins. You will notice the same separation of concerns between `catalog`, `catalog-react`, `catalog-common`, etc. 

> doing it here, i could install only the @procore-oss/backstage-plugin-announcements and be able to translate.

Yes, but the export comes from the `announcements-react` plugin, so that others can use it without needing to rely on the `announcements` frontend plugin.

Reply:
@gabrielcbe - just made this realization. Aren't you supposed to use the `useAnnouncementsTranslation` hook directly?

Reply:
Hey, I'll make these changes tomorrow when I have my computer on me.
And as for the later question, we don't use the hook directly, in our backstage instance where the announcements is installed, we get the translationRef and use it to pass it the values in Portuguese (in our case) for each key in the translationRef that comes from your plugin. Here are the docs, https://backstage.io/docs/plugins/internationalization/

Reply:
@kurtaking, just pushed a fix removing the export.
Also renamed the translationRef for a more explicit name for announcements plugin